docs/community.md

```diff
@@ -26,3 +26,13 @@ see our [dedicated page](contributing.md)
   build together 💪.
 
   To get started, jump to [GitHub Discussion](https://github.com/wise-agents/wise-agents/discussions) and start your first thread.
+
+## 👋 Zulip Chat!
+  We’re using Zulip Chat as another place to connect with other members of our community. 
+  This is a good place to connect directly and quickly with other community members.
+  You can ask question and start discussions there, just remember it's not a support chat, but just a community chat.
+  Of the discussions go longer nad need to be formalized or just recorded for the future, other community memebers can ask you to switch to GitHub Discussion.
```

Comment:
Maybe instead of "Of the discussions go longer nad need to be formalized or just recorded for the future, other community memebers can ask you to switch to GitHub Discussion.", we can use:

"For more involved discussions or to help keep track things, other community members might ask you to start a GitHub Discussion."